cypress/support/step-definitions/confirm-steps.js

```diff
@@ -1,22 +1,23 @@
 import { visitComponentUrl } from '../helper';
 import {
   asSelect, sizeSelect, subtextInput, titleInput, heightInput, labelInput,
+  disableEscKeyCheckbox, backgroundUILocator,
 } from '../../locators';
 import {
-  openButton, dialogInnerContent, dialogTitle, cancelLabel, confirmLabel, backgroundUICheckbox,
-  backGroundUILocator, disableEscKeyCheckbox, dialogPreview, showCloseIconCheckbox,
-  closeIconButton, dialogSubtitle, stickyFormFooterCheckbox, childrenTextArea,
+  openButton, dialogTitle, cancelLabel, confirmLabel,
+  dialogPreview, showCloseIconCheckbox, closeIconButton,
+  dialogSubtitle, stickyFormFooterCheckbox,
+  childrenArea, confirmButton, cancelButton,
 } from '../../locators/confirm';
 
-const CARBON_DIALOG_PREFIX = 'carbon-dialog__dialog--';
-const STICKY_FORM_FOOTER_PARAMETER = 'carbon-dialog__dialog--sticky-form-footer';
+const STICKY_FORM_FOOTER_PARAMETER = 'TODO';
```

Comment:
TODO? ;-)

Reply:
todo - it means it should added, this parameter is used in step for stickyFormFooter - the functionality exists, but doesn't work on Storybook / Carbon demo site.

Reply:
please remove STICKY_FORM_FOOTER_PARAMETER, add comment to step definitions

Reply:
removed